tests/app/submitter/test_converter.py

```diff
@@ -399,6 +399,55 @@ def test_converter_checkboxes_with_q_codes_and_other_value(self):
             self.assertEqual(answer_object['data']['1'], 'Ready salted')
             self.assertEqual(answer_object['data']['4'], 'Bacon')
 
+    def test_converter_q_codes_for_empty_strings(self):
+        with self.application.test_request_context():
+            routing_path = [Location(group_id='favourite-food', group_instance=0, block_id='crisps')]
+            answers = [create_answer('crisps-answer', '', group_id='favourite-food', block_id='crisps')]
+            answers += [create_answer('other-crisps-answer', 'Ready salted', group_id='favourite-food', block_id='crisps')]
+
+            questionnaire = {
```

Comment:
Maybe, it was done to be consistent with what was already there. I think given we need to spend some time increasing and improving the downstream test coverage, we should review all of the tests then.